nearlib/dev.js

```diff
@@ -12,6 +20,28 @@ module.exports = {
         await this.getOrCreateDevUser(near);
         return near;
     },
+    /**
```

Comment:
let's have one method which takes an object with options. Otherwise we might have combinatorial explosion of different `createXXX` methods over time.

Reply:
actually a good point, there need to be docs for individual options ideally, a-la:
https://github.com/nearprotocol/nearcore/blob/master/nearlib/near.js#L193

this doesn't have to be blocking PR, but I don't want it to linger for weeks

Reply:
Added some documentation for current args.